tests/lit-tests/lit.cfg

```diff
@@ -61,6 +61,16 @@ elif windows_enabled == "OFF":
 else:
     sys.exit("Cannot parse windows_enabled: " + windows_enabled)
 
+# PS target OSes are enabled
+ps_enabled = lit_config.params.get('ps_enabled')
```

Comment:
I don't see usage of `ps_enabled` in lit tests.

Reply:
Because I forgot to add it under git.